src/Compiler/SyntaxTree/SyntaxTrivia.fsi

```diff
@@ -44,9 +44,9 @@ type CommentTrivia =
     | LineComment of range: range
     | BlockComment of range: range
 
-/// Represents additional information for ParsedImplFileInput
+/// Represents additional information for ParsedInput
 [<NoEquality; NoComparison>]
-type ParsedImplFileInputTrivia =
+type ParsedInputTrivia =
```

Comment:
While looking into the necessary changes in Fantomas to accomodate this PR, I found that unifying these two types simplifies a lot of code in both the compiler and fantomas.core.